instrument/src/main/java/edu/berkeley/cs/jqf/instrument/tracing/FastCoverageSnoop.java

```diff
@@ -0,0 +1,45 @@
+package edu.berkeley.cs.jqf.instrument.tracing;
+
+import janala.instrument.FastCoverageListener;
+
+public class FastCoverageSnoop {
```

Comment:
Will `FastCoverageSnoop` also track the coverage of generators? I think this behavior is slightly different from JQF but I don't think it will affect the performance of the fuzzer.

Reply:
I was not trying to change which classes had coverage probes inserted. As far as I am aware, both the existing coverage implementation and mine will not track coverage in classes defined in `janala.excludes`.